### PR TITLE
Lat Arriving Metrics Doc changes

### PR DESCRIPTION
**Pull request by angadrandhawa1**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by angadrandhawa1** [comment]
Do Not auto merge.

File: docs/send-data/hosted-collectors/http-source/otlp.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ To configure an OTLP/HTTP Source:
 1. [Create any Processing Rules](/docs/send-data/collection/processing-rules/create-processing-rule) you'd like for the OTLP/HTTP Source. <br/>  ![OTLP processing rules.png](/img/send-data/OTLP-processing-rules.png)
 1. When you are finished configuring the Source, click **Save**.
 
+:::note
+1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent HTTP Endpoint have appropriate timestamps.
```
Comment (angadrandhawa1) [inline docs/send-data/hosted-collectors/http-source/otlp.md:42]:
```suggestion
1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to OTLP Endpoint have appropriate timestamps.
```

File: docs/send-data/hosted-collectors/http-source/logs-metrics/upload-metrics.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ sidebar_label: Uploading Metrics
 After you have added an [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source) to a [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector) you can begin uploading data. You can upload both logs and metrics to the same HTTP source, however not in the same HTTP request. This document provides instructions on uploading metrics, if you are uploading logs see [Upload Logs to an HTTP Source](/docs/send-data/hosted-collectors/http-source).
 
 :::note
-Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics. 
+1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent HTTP Endpoint have appropriate timestamps.
```
Comment (angadrandhawa1) [inline docs/send-data/hosted-collectors/http-source/logs-metrics/upload-metrics.md:10]:
```suggestion
1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to HTTP Endpoint have appropriate timestamps.
```

File: docs/send-data/hosted-collectors/http-source/logs-metrics/index.md [hunk 3]
```diff
@@ -47,6 +47,12 @@ To configure an HTTP Logs and Metrics Source:
 1. When you are finished configuring the Source, click **Save**.
 1. When the URL associated with the source is displayed, copy the URL so you can use it to upload data.<br/> ![img](/img/send-data/http-source-address.png)
 
+:::note
+1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to HTTP Endpoint have appropriate timestamps.
+2. Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics. 
+:::
+
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/http-source/logs-metrics/index.md:54]:
```suggestion
:::note
* Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to HTTP Endpoint have appropriate timestamps.
* Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics. 
:::
```

File: docs/send-data/hosted-collectors/http-source/logs-metrics/upload-metrics.md [hunk 4]
```diff
@@ -7,7 +7,8 @@ sidebar_label: Uploading Metrics
 After you have added an [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source) to a [Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector) you can begin uploading data. You can upload both logs and metrics to the same HTTP source, however not in the same HTTP request. This document provides instructions on uploading metrics, if you are uploading logs see [Upload Logs to an HTTP Source](/docs/send-data/hosted-collectors/http-source).
 
 :::note
-Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics. 
+1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to HTTP Endpoint have appropriate timestamps.
+2. Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/http-source/logs-metrics/upload-metrics.md:11]:
```suggestion
* Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to HTTP Endpoint have appropriate timestamps.
* Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see [Data Limits for Metrics](/docs/metrics/manage-metric-volume/data-limits-for-metrics).
```

File: docs/send-data/hosted-collectors/http-source/otlp.md [hunk 5]
```diff
@@ -38,6 +38,11 @@ To configure an OTLP/HTTP Source:
 1. [Create any Processing Rules](/docs/send-data/collection/processing-rules/create-processing-rule) you'd like for the OTLP/HTTP Source. <br/>  ![OTLP processing rules.png](/img/send-data/OTLP-processing-rules.png)
 1. When you are finished configuring the Source, click **Save**.
 
+:::note
+1. Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to OTLP Endpoint have appropriate timestamps.
+2. Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see Data Limits for Metrics.
+:::
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/http-source/otlp.md:44]:
```suggestion
:::note
* Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to OTLP Endpoint have appropriate timestamps.
* Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see [Data Limits for Metrics](/docs/metrics/manage-metric-volume/data-limits-for-metrics).
:::
```

File: docs/send-data/installed-collectors/sources/streaming-metrics-source.md [hunk 6]
```diff
@@ -38,6 +38,10 @@ Perform these steps on each host from which you want to collect metrics:
 
 1. Click **Save**.
 
+:::note
+Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Please make sure that the Metrics sent to the Streaming Metrics source have appropriate timestamps.
```
Comment (jpipkin1) [inline docs/send-data/installed-collectors/sources/streaming-metrics-source.md:42]:
```suggestion
Metrics reported with a timestamp older than 24 hours ago or newer than 24 hours in the future from the time they are reported are dropped. Make sure that the Metrics sent to the Streaming Metrics source have appropriate timestamps.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@angadrandhawa1, I have just a few updates for you. Click the **Commit suggestion** button to make the updates. Then I will approve the PR.

In the future, if you have repeating information that will appear in multiple articles, consider [reusing content](https://help.sumologic.com/docs/contributing/style-guide/#reusing-content).

**Comment by aszczepaniksumo** [comment]
Let's close this PR as a duplicate, as I moved changes from this PR to a different one (which includes more doc updates & release notes).
https://github.com/SumoLogic/sumologic-documentation/pull/2804

**Comment by jpipkin1** [comment]
Closing this PR because it it replaced by PR #2804 per @aszczepaniksumo.

**Comment by jpipkin1** [comment]
Closing this PR because it it replaced by PR #2804 per @aszczepaniksumo.

**Pull request closed without merging**